client/components/cloud/project/compute/volumes/volume/volume.factory.js

```diff
@@ -75,71 +76,51 @@ angular.module("managerApp").factory('CloudProjectComputeVolumesVolumeFactory',
             });
         };
 
+        /* jshint ignore:end*/
         /**
          *  [API] Get additional informations about volume (price)
-         *  Create a gbPrices attribute like { 'region' : { 'volumeType' : { 'price' : {}, 'monthlyPrice' : {} }  } }
+         *  Create a volumePricesMap attribute like { 'planCode' : { 'price' : {} } }
          */
         VolumeFactory.prototype.getFullInformations = function () {
             var self = this;
-
-            return OvhApiCloudPrice.Lexi().query().$promise.then(function (prices) {
-
-                // group volume prices by region to avoid making a request to get regions list
-                var prices = _.groupBy(prices.volumes, 'region'),
-                    tmpRegionTypePrice;
-
-                // init gb prices attribute
-                self.gbPrices = {};
-
-                // prices is an object with regions as keys
-                angular.forEach(prices, function (regionPrices, region) {
-                    self.gbPrices[region] = {};
-                    // try to find a price with type list
-                    angular.forEach(CLOUD_VOLUME_TYPES, function (volType) {
-                        tmpRegionTypePrice = _.find(regionPrices, {
-                            volumeName : 'volume.' + volType
-                        });
-
-                        if (tmpRegionTypePrice) {
-                            self.gbPrices[region][volType] = {
-                                price : tmpRegionTypePrice.price,
-                                monthlyPrice : tmpRegionTypePrice.monthlyPrice
-                            };
-                        }
-                    });
-                });
-            });
+            OvhCloudPriceHelper.getPrices(this.serviceName).then(response => {
+                self.volumePricesMap = response;
+            }).catch(err => console.warn(err));
```

Comment:
once again.. I'm not sure about this console.warn thing.  It doesn't give any feedback to the users OR the dev if it happens in prod.  We should talk, I want to understand why you do that I think I'm missing something

Reply:
put warn because there was warn in legacy code ^^